addons/web/controllers/main.py

```diff
@@ -1022,10 +1022,22 @@ def test_mobile_suite(self, mod=None, **kwargs):
     def benchmarks(self, mod=None, **kwargs):
         return request.render('web.benchmark_suite')
 
+    @http.route('/web/bundle/<string:bundle_name>', type='http', auth="user", methods=["GET"])
+    def bundle(self, bundle_name):
+        """
+        Request the definition of a bundle, including its javascript and css bundled assets
+        """
+        files = request.env["ir.qweb"]._get_asset_nodes(bundle_name, debug=request.session.debug, js=True, css=True)
+        data = json.dumps([{
+            "type": tag,
+            "src": attrs.get("src") or attrs.get('href'),
+        } for tag, attrs, _ in files])
+        return request.make_response(data, [('Content-Type', 'application/json')])
+
 
 class Proxy(http.Controller):
 
-    @http.route('/web/proxy/post/<path:path>', type='http', auth='user', methods=['GET'])
+    @http.route('/web/proxy/post/<path:path>', auth='user', methods=['GET'])
```

Comment:
Not sure why we need this diff but since http is the default type I don't really mind it

Reply:
damn it, I didn't mean to change that. It's the second time I touch this line by accident. 
Thanks for telling me. I'll book an appointment with an eye-doctor 😄